hedera-node/hedera-mono-service/src/test/java/com/hedera/node/app/service/token/util/UtilsConstructorTest.java

```diff
@@ -23,7 +24,8 @@
 import org.junit.jupiter.api.Test;
 
 class UtilsConstructorTest {
-    private static final Set<Class<?>> toBeTested = new HashSet<>(Arrays.asList(AliasUtils.class));
+    private static final Set<Class<?>> toBeTested =
```

Comment:
Sonar only counts coverage for a class based on the unit tests **within its module**.

So we need to add `FeeUtils.class` to [this list](https://github.com/hashgraph/hedera-services/blob/develop/hapi-utils/src/test/java/com/hedera/services/UtilsConstructorTest.java#L40) to get its private constructor covered.

Reply:
Wow. Thanks !